File: intellij-lsp-dotty/src/DottyLanguageExtension.java
import com.github.gtache.ServerDefinitionExtensionPoint;

public class DottyLanguageExtension extends ServerDefinitionExtensionPoint {

    public static DottyLanguageExtension INSTANCE;

    public DottyLanguageExtension() {
        super("scala", "ch.epfl.lamp:dotty-language-server_0.3:0.3.0-RC2", "dotty.tools.languageserver.Main", new String[]{"-stdio"});
    }
}

File: intellij-lsp-dotty/src/DottyLanguageExtension.scala
import com.github.gtache.{LanguageServerDefinition, ArtifactLanguageServerDefinition}


object DottyLanguageExtension extends ArtifactLanguageServerDefinition("scala", "ch.epfl.lamp:dotty-language-server_0.4:0.4.0-RC1", "dotty.tools.languageserver.Main", Array[String]("-stdio")) {
  def register(): Unit = LanguageServerDefinition.register(this)
}